panther_bringup/config/imu_config.yaml

```diff
@@ -1,18 +1,25 @@
 ImuFilterNodelet:
+  gain: 0.1
+  zeta: 0.0
+  mag_bias_x: 0.0
+  mag_bias_y: 0.0
+  mag_bias_z: 0.0
+  orientation_stddev: 1e-2
   publish_tf: false
   use_mag: false
-  use_magnetic_field_msg: false
+  use_magnetic_field_msg: true
```

Comment:
Will magnetometer data be useful in our case? IMU is quite close to motors and additionally it closed in the robot body, so I think that it won't be really accurate.

Reply:
this change only message type to correct one, dont publish mag data (`use_mag: false` is still on false)